### ISSUE TITLE
downlink how you send to nodes from mqtt or ttn?

### ISSUE THREAD

**Issue by halukmy** [things4u/ESP-1ch-Gateway-v5.0#2]
hello all, i just collect data when node send to gateway(esp12e)  via mqtt parsing so its great i also send downlink via mqtt over ttn network and i can follow traffic on application data 

but when i send data on mqtt network via mqtt i see on application traffic area but coulndt see on traffic on gateway layout also no seen on arduino debug console

any tip?

thanks

**Comment by zenghi** [comment]
HI halukmy I successfully use the downlink ，you can view PULL REQUESTS.

**Comment by halukmy** [comment]
@zenghi thanks so much I been waiting long so you have any demo? 

Now mqtt works our nodemcu gateway? That's brilliant! Thanks much

**Comment by zenghi** [comment]
![image](https://user-images.githubusercontent.com/6140297/34036636-d77e93b2-e1c0-11e7-93a5-bc1438dd2732.png)
![image](https://user-images.githubusercontent.com/6140297/34036662-eb276d30-e1c0-11e7-9f8c-60e0556e7980.png)
![image](https://user-images.githubusercontent.com/6140297/34036895-a55505a0-e1c1-11e7-9b9a-89842b5d4bd5.png)
![image](https://user-images.githubusercontent.com/6140297/34037030-38a99302-e1c2-11e7-9874-68585041e83c.png)


Use  ESP8266 to do single-channel receive and transmit,terminal part can see ACK,Platform-side can see the datat storage.

**Comment by halukmy** [comment]
Like a dream! Thanks much!

What's your lmic source and version?

**Comment by zenghi** [comment]
SX1278 and esp8266，SX1276 I think the same applies
The terminal uses STM32L151 and SX1278
Maybe I did not understand you very well,hope this helps.

**Comment by halukmy** [comment]
Pardon me @zenghi 

i mean, what you use for nodes? for example

https://github.com/lyusupov/SoftRF/tree/master/software/firmware/source/libraries/arduino-lmic 
this lib you advices?

and second question,

is it possible message without gateway with secure? did you work any on it?

**Comment by zenghi** [comment]
I'm sorry, I see your message now。
I use the following lib
https://github.com/Lora-net/LoRaMac-node

The second question, No corresponding work

**Comment by halukmy** [comment]
im using www.dorji.com/docs/data/DRF1278F.pdf with lmic 1.5 , is my dorji work with LoraMac-node? i cant see any examples ino in semtech repo:(

**Comment by zenghi** [comment]
If possible, I can give you a set of hardware. But the logistics cycle will be longer.

**Comment by halukmy** [comment]
@zenghi you serious? that would be awesome!

**Comment by halukmy** [comment]
im trying to sx1278 as gateway btw

**Comment by zenghi** [comment]
Yes, at present, the complete circuit is being debugged, and once the success will be sent to a device to you, please send your address to my private mailbox.

**Comment by halukmy** [comment]
Thanks @zenghi , i just send mail to 309966755@qq.com from haluky@live.com

**Comment by halukmy** [comment]
@zenghi hi Zenghi,

do you have Class-C support for this channel for limic ?

**Comment by schwiderke92** [comment]
downlink is work?

**Comment by richard-ubi** [comment]
I have encountered such a problem, I sent the data to LoRaServer only uplink, downlink error
![8 4I2GXL1JH%YL@2O @KUWG](https://user-images.githubusercontent.com/47618847/60012341-45246600-96ae-11e9-9712-916e9796d53e.png)
